client/task-list/tasks/payments/stripe.js

```diff
@@ -167,15 +167,21 @@ class Stripe extends Component {
 	validateManualConfig( values ) {
 		const errors = {};
 
-		if ( values.publishable_key.match( /^pk_live_/ ) === null ) {
+		if (
+			values.publishable_key.match( /^pk_(live|test)_[a-zA-Z0-9_]+/ ) ===
+			null
+		) {
 			errors.publishable_key = __(
-				'Please enter a valid publishable key. Valid keys start with "pk_live".',
+				'Please enter a valid publishable key.',
```

Comment:
Nice catch Joel!
As this problem only happens on mobile, what do you think about adding a CSS code instead of cutting the text to solve it?
It should look like this code:
```
@include breakpoint( '<782px' ) {
	margin-bottom: $gap-small + $gap * 2 !important;
}
```
And probably it should be added [here](https://github.com/woocommerce/woocommerce-admin/blob/main/client/layout/style.scss#L95)

Reply:
Good call @octaedro ! That CSS snippet works great. As far as I can tell that `<Form>` component doesn't support the `className` property, or allow for custom classes. I could always add a wrapper but that doesn't seem ideal.

That said, I think you could make the argument that we should leave this CSS tweak as is without narrowing it to a particular form. I poked around and saw a couple instances of validation messages on mobile being a bit crowded:

![image](https://user-images.githubusercontent.com/444632/94855847-fc487a80-03e3-11eb-863c-e8702c7d5fd6.png)

Although not entirely overlapping.

Reply:
@joelclimbsthings fwiw, the <Form> component doesn't actually render it's own component, it clones the children and renders them so you could add the class to a wrapper inside <Form>

Reply:
> That said, I think you could make the argument that we should leave this CSS tweak as is without narrowing it to a particular form. I poked around and saw a couple instances of validation messages on mobile being a bit crowded:

Yes I think if you're easily uncovering other issues it may be safer to truncate the copy.